src/components/radiobutton/radiobutton.component.html

```diff
@@ -1,19 +1,20 @@
-<label [attr.for]="(id || _radioButtonId) + '-input'"
-       class="ux-radio-button"
-       [class.ux-radio-button-checked]="value === option"
-       [class.ux-radio-button-simplified]="simplified"
-       [class.ux-radio-button-disabled]="disabled"
-       [class.ux-radio-button-focused]="_focused">
-
+<label
+    [attr.for]="(id || _radioButtonId) + '-input'"
+    class="ux-radio-button"
+    [class.ux-radio-button-checked]="value === option"
+    [class.ux-radio-button-simplified]="simplified"
+    [class.ux-radio-button-disabled]="disabled"
+    [class.ux-radio-button-focused]="_focused"
+>
     <div class="ux-radio-button-container">
-
-        <input class="ux-radio-button-input"
+        <input
+            class="ux-radio-button-input"
             uxFocusIndicator
             type="radio"
             [id]="(id || _radioButtonId) + '-input'"
             [checked]="value === option"
             [disabled]="disabled"
-            [tabindex]="tabindex || value === option ? 0 : -1"
+            [readonly]="readonly"
```

Comment:
`readonly` is not a valid attribute on `input type="radio"`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly